extensions/resteasy-reactive/rest-client/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientRecorder.java

```diff
@@ -1,17 +1,28 @@
 package io.quarkus.rest.client.reactive.runtime;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
 
 import org.eclipse.microprofile.rest.client.spi.RestClientBuilderResolver;
 
+import io.quarkus.runtime.ShutdownContext;
 import io.quarkus.runtime.annotations.Recorder;
+import io.vertx.core.http.HttpClient;
 
 @Recorder
 public class RestClientRecorder {
     private static volatile Map<String, String> configKeys;
     private static volatile Set<String> blockingClassNames;
 
+    private static volatile Map<String, String> blockingMethods;
```

Comment:
I can't see where this one is used.

Reply:
They aren't, but that's for another PR since it was already there